release-engineering/role-handbooks/branch-manager.md

```diff
@@ -887,3 +889,17 @@ See the branch management process prior to v1.12 when `anago` was still used.
 [release-branch-job-creation]: https://git.k8s.io/test-infra/releng/README.md
 [release-managers]: /release-managers.md#release-managers
 [release-managers-group]: https://groups.google.com/a/kubernetes.io/forum/#!forum/release-managers
+
+# Visual Release Cut Process
+
+The diagram below shows the actions needed to cut a Kubernetes release
+
+```mermaid
+flowchart TD
+    Start --> issue["Create a release cut issue on GitHub"] -->|"update along the way"| issue
+    issue --> thread["Create a Slack Thread in #release-management \n and cc release-managers"] -->|"update along the way"| thread
+    thread --> mock["Mock Release Cut"]
+    mock --> staging["Kick off Release"]
+    staging --> artifact_promotion["Promote and Sign Artifacts"]
+    artifact_promotion --> release["Release New Kubernetes Version"]
+```
```

Comment:
- Then we have to get in contact with the Google build admin to cut the debs/rpms
- Then we notify the channel that the release has been cut
- Then we send the announcement via krel
- Then we close the issue if no follow-up work is required